worldengine/cli/main.py

```diff
@@ -104,13 +103,11 @@ def generate_plates(seed, world_name, output_dir, width, height,
     filename = '%s/plates_%s.png' % (output_dir, world_name)
     # TODO calculate appropriate sea_level
     sea_level = 1.0
-    draw_simple_elevation_on_file(world.elevation['data'], filename, width,
-                                  height, sea_level)
+    draw_simple_elevation_on_file(world, filename, None)
```

Comment:
Ok. I had been working on something to change the ocean-level but it turned out that passing a different value to platec (currently it always assumes 65% of the surface to be covered with ocean) resulted in ugly maps. So let's just do it your way and come back to this in the distant future. (But the dead variable should be taken care of by somebody to make pyflakes happy again.)